src/platform/testing/e2e/cypress/support/commands/axeCheck.js

```diff
@@ -33,6 +33,10 @@ Cypress.Commands.add('axeCheck', (context = 'main') => {
 
   const options = {
     includedImpacts: ['critical'],
```

Comment:
Do we still need this setting? I would've expected this line to prevent errors on anything below critical severity, but from your example, maybe it doesn't affect anything. In which case, it seems like removing it would prevent confusion down the road.

Reply:
@1Copenut do you have any input here?

Reply:
The `includedImpacts` key and array need to be removed. This limits what level of violations we're looking for, which is not what we want. The added lines below frame the proper bounds checking.